x/slashing/spec/01_concepts.md

```diff
@@ -16,6 +16,8 @@ For each of these validators we keep a `ValidatorSigningInfo` record that contai
 information partaining to validator's liveness and other infraction related
 attributes.
 
+Thoughts: ValidatorSigningInfo is as part of slashing module?
```

Comment:
This is done for liveness tracking.  It keeps track of how many of the last n blocks each validator signs on, and if below a threshold, slashes and jails them.